changelog.d/290.change.rst

```diff
@@ -0,0 +1 @@
+Added ``attr.field_dict()`` to return an ``OrderedDict` of ``attrs`` attributes for a class, whose keys are the attribute names.
```

Comment:
I think we shouldn’t speak of ``OrderedDict`` since that’s not true. Just say “ordered dict” or “ordered dictionary”. I’m not gonna point it out in every instance of your PR to avoid clutter, but please get rid of it. :)